kura/org.eclipse.kura.nm/src/main/java/org/eclipse/kura/nm/configuration/NetworkProperties.java

```diff
@@ -21,7 +21,7 @@
 
 public class NetworkProperties {
 
-    private Map<String, Object> properties;
+    private final Map<String, Object> properties;
 
```

Comment:
Please put this class in a different package (i.e. org.eclipse.kura.nm) since this is common for status and configuration.